proto/regen/ecocredit/v1alpha1/events.proto

```diff
@@ -28,8 +28,8 @@ message EventCreateBatch {
   // issuer is the account address of the issuer of the credit batch.
   string issuer = 3;
 
-  // total_units is the total number of units in the credit batch.
```

Comment:
why renaming to amount? to be more consistent with x/bank?

Reply:
Yes, and @aaronc suggested this would be a more suitable name